app/styles/themes/white-label/_theme.overrides.scss

```diff
@@ -56,26 +70,255 @@
   .gothamist-letters {
     fill: rgb(var(--color-black));
   }
+
   .gothamist-buildings {
-    fill: rgb(var(--color-white));
+    fill: rgb(var(--color-black));
   }
 }
 
+// Footer
+.c-main-footer {
+  background-image: rgb(var(--color-cool-white));
+  background: rgb(var(--color-cool-white));
+  color: rgb(var(--color-black));
 
+  a,
+  a:visited,
+  a:active {
+    color: rgb(var(--color-ceramic-blue));
+    text-decoration: none;
+
+    &:hover {
+      color: rgb(var(--color-ceramic-blue));
+      text-decoration: none;
+    }
+  }
 
-.c-main-footer {
   .gothamist-letters {
-    fill: rgb(var(--color-white));
+    fill: rgb(var(--color-black));
   }
+
   .gothamist-buildings {
-    fill: rgb(var(--color-dark-gray));
+    fill: rgb(var(--color-black));
+  }
+
+  .c-main-footer__signup svg {
+    fill: rgb(var(--color-white));
+  }
+
+  .c-newsletter-form__text p,
+  .c-newsletter-form__terms p {
+    color: rgb(var(--color-white));
+  }
+
+  .c-newsletter-form__terms-checkbox a,
+  .c-newsletter-form__terms-checkbox a:visited,
+  .c-newsletter-form__terms-checkbox a:active {
+    background: transparent;
+    color: rgb(var(--color-white));
+
+    &:hover {
+      color: rgb(var(--color-white));
+    }
   }
 }
 
+// Typography
+.c-basic__heading {
+  @include typeface(header, 13);
+  @include media(">=medium") {
+    @include typeface(header, 15);
+  }
+}
+
+.c-block__title,
+.c-block--horizontal--large .c-block__title {
+  @include media(">medium") {
+    @include typeface(header, 8);
+  }
+
+  a,
+  a:visited,
+  a:active {
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: none;
+    }
+  }
+}
+
+.o-rte-text > dl dd,
+.o-rte-text > dl dt,
+.o-rte-text > ol li,
+.o-rte-text > ul li,
+.o-rte-text > p {
+  @include typeface(body, 5);
+}
+
+.o-rte-text > h1, .o-rte-text > h2, .o-rte-text > h3, .o-rte-text > h4, .o-rte-text > h5, .o-rte-text > h6 {
```

Comment:
separate class/tag specifiers onto the their own lines like you did just above here

Reply:
good catch, I copied and pasted this from the browser!